pycytominer/annotate.py

```diff
@@ -113,6 +117,7 @@ def annotate(
                 left_on=external_join_left,
                 right_on=external_join_right,
                 how="left",
+                suffixes=[None, "_external"],
```

Comment:
I agree a duplicate column check would be nice, but this sounds good to me as is.

Reply:
I'm not sure I understand what you mean with this comment.